protocol/avdl/user.avdl

```diff
@@ -67,10 +67,16 @@ protocol user {
   /**
     The list-tracking function get verified data from the tracking statements
     in the user's own sigchain.
+
+    The *JSON functions are unfortunate, but the current implementation includes
+    a lot more than the non-JSON option, so leaving it like this...
     */
   array<UserSummary> listTracking(int sessionID, string filter);
   string listTrackingJSON(int sessionID, string filter, boolean verbose);
-
+  array<UserSummary> listTrackingForUID(int sessionID, string filter, UID uid);
```

Comment:
In general, it would be nice if all parts of the CLI used the same convention, that whenever you enter a user, you can enter any assertion.  There remains the issue of `a@twitter+b@github`-style assertions, which are broken, but we can get to that eventually.